package.json

```diff
@@ -47,7 +47,7 @@
     "@octokit/webhooks": "^12.0.10",
     "@probot/get-private-key": "^1.1.2",
     "@probot/octokit-plugin-config": "^2.0.1",
-    "@probot/pino": "^2.5.0",
+    "@probot/pino": "^3.0.0",
```

Comment:
why does this add 1k lines of package-lock?

Reply:
Because of updated dependencies, specifically `@sentry/node` v8
It now pulls in opentelemetry